app/experimenter/experiments/management/commands/generate_dashboards.py

```diff
@@ -14,10 +15,26 @@ class Command(BaseCommand):
     help = 'Generates Redash dashboards'
 
     def generate_dashboards(self):
-        relevant_experiments = Experiment.objects.filter(
-            status__in=(
-                Experiment.STATUS_LAUNCHED,
-                Experiment.STATUS_COMPLETE))
+        recent_changelog_complete = ExperimentChangeLog.objects.filter(
```

Comment:
You could combine all of these together so that it's a single query on the Experiment model using Q objects:

https://docs.djangoproject.com/en/1.11/topics/db/queries/#complex-lookups-with-q

So then it would look something like

        Experiment.objects.filter(
            # Recently completed
            Q(
                status=Experiment.STATUS_COMPLETE,
                changes__old_status=Experiment.STATUS_LAUNCHED,
                changes__new_status=Experiment.STATUS_COMPLETE,
                changes__changed_on__gte=.....
             ) |
            # In flight
            Q(
                status=Experiment.STATUS_LAUNCHED,
            ) |
            ...
        ).distinct()

etc   

Also I would suggest moving all of that into the model manager itself here:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/models.py#L14

as a method called like 'should_update_dashboard' or something

And then put tests in the model manager tests to test the different filtering clauses, and then in your actual dashboard management command test, just assume that an experiment returned by that method should be updated and then the management command test doesn't need to know or care about that cases of experiment states to update.